pkg/vals/secret_stores.go

```diff
@@ -0,0 +1,37 @@
+package vals
```

Comment:
what does this word "vals“ short for? "values"? If this pkg is designed for SecretStores, maybe you can add a doc.go file or change it to a more understandable one

Reply:
ok, I will add a doc.go